modules/apps/collaboration/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/select_file_entry.jsp

```diff
@@ -102,11 +102,11 @@ dlSearchContainer.setResults(foldersAndFileEntriesAndFileShortcuts);
 								Map<String, Object> data = new HashMap<String, Object>();
 
 								data.put("entryid", fileEntry.getFileEntryId());
-								data.put("entryname", fileEntry.getTitle());
+								data.put("entryname", HtmlUtil.unescape(fileEntry.getTitle()));
```

Comment:
This change looks inconsistent with https://github.com/sergiogonzalez/liferay-portal/pull/4251/commits/b168c253b5b6e5343e4bbf69ce7dca83e8b19566#diff-6ac3ea56964b221131492996ff7ecc85R127. In this one we are unescaping an escape entry, but in https://github.com/sergiogonzalez/liferay-portal/pull/4251/commits/b168c253b5b6e5343e4bbf69ce7dca83e8b19566#diff-6ac3ea56964b221131492996ff7ecc85R127 we are using the escaped entry. Seems like there's a bug somewhere.

Reply:
Hello @sergiogonzalez ,

After checking the suggestion, I am not completely sure whether it's a bug, but more on how the implementation is being done. fileEntry.getTitle() will eventually have to be unescaped before being used in edit_file_short.jsp because select_file_entry.jsp (https://github.com/jonathanmccann/liferay-portal/blob/b168c253b5b6e5343e4bbf69ce7dca83e8b19566/modules/apps/collaboration/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/select_file_entry.jsp#L61) uses cast_results.jspf to return escaped models of fileEntry and fileShortcut.